maas-schemas-ts/README.md

```diff
@@ -1,6 +1,6 @@
 # MaaS Schemas TypeScript Support
 
-This repostory contains [io-ts](https://github.com/gcanti/io-ts) validators that match the [JSON Schema](https://json-schema.org/) definitions from the [maas-schemas](https://github.com/maasglobal/maas-schemas) package. The validators work with any JavaScript compatible language. However, the main benefit of io-ts validators is their compatibility with TypeScript's static type system.
+This repostory contains [io-ts](https://github.com/gcanti/io-ts) validators that match the [JSON Schema](https://json-schema.org/) definitions from the [maas-schemas](https://github.com/maasglobal/maas-schemas/tree/develop/maas-schemas) package. The validators work with any JavaScript compatible language. However, the main benefit of io-ts validators is their compatibility with TypeScript's static type system.
```

Comment:
Maybe don't link to develop specifically. Perhaps it is not necessary to have a link at all since they are now in the same monorepo.

Reply:
Good point :+1: replaced with relative link